### PR TITLE
refactor tool definition

### PR DESCRIPTION
**Pull request by NiklasTR**
## Motivation
Based on: 
https://www.notion.so/labdao/PLEX-0-5-Let-s-clean-up-to-make-things-more-useful-21fd329e1bc241039d57932f7af5f7be?pvs=4

TLDR: tools (eventually chains of tools) should be easier to define, have explicit inputs/outputs and define to best practices (CWL) if there is no good reason to deviate from it. 

## Changes introduced:

- [x] rename "app" to "tool" for consistency with the ontology
- [x] introduce a set of example tool definitions: equibind and simpdock - with the focus being on simpdock, a testfile
- [x] change the install.sh to not download the individual instruction.jsonl and app config, but download the directory of json tool definitions. 
- [ ] rewrite the tests for test_instruction.go -> Instruction is where the tool definition and data definitions (data includes input parameters) are integrated to then eventually run jobs. 
- [ ] adapt and simplify instruction.go -> the main idea is that the tool definition (generated through a reader function in tool.go -currently implemented in ligase.go-) *and* the data(input + params) is used to construct a complete instruction set. 
- [ ] update the toolConfigsFilePath in main.go to list all *json* files in the config directory as possible applications
- [ ] check that InstructiontoDockerCmd and InstructiontoBacalhauCmd work properly

image

## Consequences: 
no apps but equibind will work in this implementation, and equibind will only be able to take one protein and one small molecule.

### CODE REVIEW

**Comment by thetechnocrat-dev** [comment]
stale
